### PR TITLE
RedisHandler support

### PR DESCRIPTION
**Pull request by LukaszPiechowiak**

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -286,6 +286,13 @@
  *   - host: server log host. ex: 127.0.0.1:9911
  *   - [level]: level name or int value, defaults to DEBUG
  *   - [bubble]: bool, defaults to true
+ * 
+ *  - redis:
```
Comment (roelmonnens) [inline DependencyInjection/Configuration.php]: Could you add id here

File: DependencyInjection/MonologExtension.php [hunk 2]
```diff
@@ -699,6 +699,25 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             ));
             break;
 
+        case 'redis':
+            if(class_exists('\Redis')){
```
Comment (roelmonnens) [inline DependencyInjection/MonologExtension.php]: could you check for the id-parameter here?

File: DependencyInjection/MonologExtension.php [hunk 3]
```diff
@@ -699,6 +699,25 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
...
+                $client = new Definition('\Redis');
```
Comment (roelmonnens) [inline DependencyInjection/MonologExtension.php]: could you combine my pullrequest with yours? Then we have a monologHandler for Redis and Predis that works with services as with just redis-host, port, ... etc...
https://github.com/symfony/monolog-bundle/pull/225/files

Then I can close my pullrequest.

**Comment by ruudvdd** [comment]
What needs to be done to get this pr merged?

**Comment by jcerrada** [comment]
Hi! Will this be merged in a near future?

**Comment by MichielDilissen** [comment]
+1

**Comment by gvanbeck** [comment]
+1

**Comment by JurRutten** [comment]
+1

**Comment by Seldaek** [comment]
I agree that this should be adjusted to also allow passing a service id for the redis/predis client as per @roelmonnens' comments. If an app already has a redis connection open for other purposes it's kinda silly to open a second one just for the logs.

**Comment by Seldaek** [comment]
Also if any of the +1 club wants to take this PR and do the required changes that'd be very helpful.

**Comment by ruudvdd** [comment]
@Seldaek Since I don't know how to update this pr, I have created a new one with a branch created from a more recent version of master. https://github.com/symfony/monolog-bundle/pull/292

**Comment by lyrixx** [comment]
Closing in favor of #292

**Comment by lyrixx** [comment]
Thanks @LukaszPiechowiak for the initial work

**Pull request closed without merging**